css/styles.css

```diff
@@ -0,0 +1,94 @@
+.row {
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+}
+
+img {
+  width: 100%;
+}
+
+header {
+  padding-top: 0.1px;
+  padding-bottom: 0.1px;
+  background-color: #4c2676;
+}
+
+.header {
+  color: white;
+}
+
+input {
+  margin-left: 1px;
+  margin-top: 7px;
+  width: 100%;
+}
+
+.button {
+  margin-left: 300px;
+}
+
+@media screen and (width > 0) {
+  p {
+    margin-bottom: 5px;
+  }
+
+  @media screen and (width > 424px) {
+    p {
+      margin-bottom: 6px;
+    }
+  }
+
+  @media screen and (width > 767px) {
+    p {
+      margin-bottom: 5px;
+    }
+  }
+}
+
+@media screen and (width > 567px) {
+  .column-half {
+    flex-basis: 50%;
+  }
+}
+
+@media screen and (width > 0) {
+  textarea {
+    width: 100%;
+    height: 100px;
+    margin-top: 5px;
+    margin-left: 1px;
+  }
+}
+
+@media screen and (width > 374px) {
+  textarea {
+    width: 100%;
+    height: 50px;
+    margin-top: 7px;
+  }
+}
+
+@media screen and (width > 424px) {
+  textarea {
+    width: 100%;
+    height: 50px;
+  }
+}
+
+@media screen and (width > 767px) {
+  textarea {
+    width: 100%;
+    height: 150px;
+    margin-left: 10px;
+    margin-top: 10px;
+  }
+}
```

Comment:
These are a lot of media queries. You just need to focus on the two that we provided for now which are for mobile screens and larger screen which is 
`@media screen and (min-width: 768px)`